source/User_Guide/Settings/subusers.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: page
+weight: 100
+title: Subusers
+seo:
+  title: Subusers
+  description: Create and Manage Subusers
+  keywords: Subuser settings, create Subuser, manage Subuser, delete Subuser
+navigation:
+  show: true
+---
+
+{% info %}
+If you have a **Pro plan or higher**, you may create up to 15 Subusers through your account. If you reach the maximum of 15 and require more Subusers, please use the form in the [Subuser Management]({{site.app_url}}/settings/Subusers) page to request more.
+{% endinfo %}
+
+Subusers are SendGrid accounts that belong to a parent account. They have their own permissions and credit limits, which you assign as you create the Subusers. Subusers are provided in order to help you segment your email sending and API activity. We suggest our customers create Subusers for each of the different types of emails they send. This often means one Subuser for transactional emails and one for marketing emails; however some of our customers will break up their sending across multiple Subusers in order to segment the statistics for each type or function of email they send.
+
+
+- [Create a Subuser](#-Create-a-Subuser)
+- [Manage Subusers](#-Manage-Subusers)
+- [Optional Profile Information](#-Optional-Profile-Information)
+- [Allocating or Removing Credits From a Subuser](#-Allocating-or-Removing-Credits-From-a-Subuser)
+- [Adding a whitelabel domain to a Subuser account](#-Adding-a-whitelabel-domain-to-a-Subuser-account)
+- [Monitoring Subuser email content](#-Monitoring-Subuser-email-content)
+- [Impersonating a Subuser](#-Impersonating-a-Subuser)
+- [Export Subuser Information](#-Export-Subuser-Information)
+- [Disabling a Subuser account](#-Disabling-Subuser-account)
+- [Deleting a Subuser account](#-Deleting-a-Subuser-account)
+
+{% anchor h2 %}
+Create a Subuser
+{% endanchor %}
+
+New SendGrid accounts do not come with Subusers and must be created by the parent account.
+
+*To create a Subuser:*
+
+1. Navigate to **Settings** and select **Subuser Management**. 
+1. Click **Create New Subuser**. 
+1. Enter the following required information into the corresponding fields:
+  * Username - The username your Subuser will use to send email and access the API.
+  * Email Address - The contact email address for this Subuser. This must be an active email account as SendGrid may contact a Subuser to provide support,  resolve sending and deliverability issues, or to enforce SendGrid's Terms of Service. Whenever we contact a Subuser, it is our policy to contact the    parent account email address at the same time.
+  * Password - The password your Subuser will use to access SendGrid.com.
+  * Avatar - A [gravatar](http://gravatar.com) will load when an email address is entered, making it easier to identify your Subusers in the Subuser list.
```

Comment:
I think this needs more explanation, it seems really random. "will load" is also passive, if you keeps this, just make it "loads when an..."